portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/passwordpolicies/PasswordPolicies.testcase

```diff
@@ -541,6 +541,271 @@ definition {
 			userEmailAddress = "userea@liferay.com");
 	}
 
+	@description = "This is a use case for LPS-193886 (TC-2). A new User cannot set their new password through the password setup link, if it is expired."
+	@priority = 5
+	test NewUserCannotSetNewPasswordThroughTheExpiredLink {
+		property custom.properties = "feature.flag.LPS-193884=true${line.separator}passwords.default.policy.change.required=false${line.separator}passwords.default.policy.reset.ticket.max.age=300";
```

Comment:
There is no need to add the _passwords.default.policy.change.required=false_ property as its default value is false:
https://github.com/liferay/liferay-portal/blob/master/build-test.xml#L11911
The following cases also have the same problem.

Reply:
I'll remove that custom property as it is and do a rerun.